atc/atccmd/command.go

```diff
@@ -1187,7 +1190,6 @@ func (cmd *RunCommand) constructLockConn(driverName string) (*sql.DB, error) {
 		return nil, err
 	}
 
-	dbConn.SetMaxOpenConns(1)
```

Comment:
I think this PR is good to go once this line is restored. :+1:

Reply:
done